### DISCUSSION TITLE
Is there a way to see how much of my deploy limits I have used?

### DISCUSSION THREAD

**Discussion by trevcodesjs** [vercel/vercel#4303] [General] [answered]
For example there is the limit of 160 Serverless Functions per Month. I would like to see how many Serverless Functions I have left for the month so that I can properly manage my deploys so as to not hit the limit.

**Comment by coetry** [comment, marked as answer]
@trevcodesjs We will absolutely make this available in the dashboard soon. For now, we fail builds that exceed 12 functions per deployment and display the relevant error message. If you're ever concerned about your individual account, please reach out to support@vercel.com and we'll take evaluate your account metrics.
